### DISCUSSION TITLE
Cannot start nuxt:  Cannot read properties of undefined (reading 'options')

### DISCUSSION THREAD

**Discussion by tejassyndell** [nuxt/nuxt#20956] [Questions]
I am upgrading my project from Nuxt 2 to Nuxt 3, and I have tried multiple solutions to resolve this error. However, none of them have been successful so far. 
https://prnt.sc/MEPaRApU7gf1

**Comment by warflash** [comment]
Nuxt/axios is a nuxt 2 module only. The module recommends using `$fetch` as an replacement -> https://axios.nuxtjs.org/ + https://nuxt.com/docs/getting-started/data-fetching/#usefetch

**Reply by tejassyndell** [reply]
I implemented the useFetch method, but it's not working for me. The error remains the same. Here is a sample of the method I implemented - https://prnt.sc/Edih2KXnX7U8
error - https://prnt.sc/7rTTDbNzBAPT

**Reply by warflash** [reply]
useFetch is only available within setup context iirc and can't be destructured into { fetch }, it returns data, error, pending and refresh properties. I recommend checking the [docs here](https://nuxt.com/docs/api/composables/use-fetch). The error you sent also still mentions axios somewhere so might be worth investigating where that is coming from as well
